src/OrchardCore/OrchardCore.FileStorage.AzureBlob/BlobFileStore.cs

```diff
@@ -300,20 +300,18 @@ private async Task CreateDirectoryAsync(string path)
                 await placeholderBlob.UploadAsync(stream);
             }
         }
-    }
 
-    internal static class BlogFileStoreExtensions
-    {
         /// <summary>
         /// Blob prefix requires a trailing slash except when loading the root of the container.
         /// </summary>
-        internal static string NormalizePrefix(this BlobFileStore blobFileStore, string prefix)
+        private string NormalizePrefix(string prefix)
```

Comment:
That's what I wanna asked you before, I'm wondering why it's an extension method